docs/fr/index.md

```diff
@@ -1,153 +1,143 @@
-environnement de calcul interactif pour traiter les données. Tous les serveurs
-bloc-notes ont accès à un maximum de 15CPU/48GB RAM et à un stockage à l'échelle
-de GB/TB, mais ont une interface utilisateur différente selon la version que
-vous choisissez.
+<!-- prettier-ignore -->
+!!! important "A l'attention des employés de Statistique Canada!"
+     Les utilisateurs internes de Statistique Canada peuvent commencer immédiatement sans aucune procédure d'inscription supplémentaire, il suffit de se rendre sur [https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca /).
 
-- Python, Julia et R via un **Bloc-notes Jupyter**.
-- R via **RStudio**
+<!-- prettier-ignore -->
+!!! note ""
+     <center>
+     [![Kubeflow est le cœur de l'AAW !](./images/Kubeflow.PNG)](https://kubeflow.aaw.cloud.statcan.ca/)
+     <h3>**[👉 Cliquez ici pour configurer votre compte Kubeflow ! 👈](https://kubeflow.aaw.cloud.statcan.ca/)**</h3>
+     </center>
 
-### Bloc-notes Jupyter pour `Python`, `Julia`, ou `R`
+**[Kubeflow](1-Experiments/Kubeflow/)** est une plate-forme open source puissante et flexible qui permet une exploitation dynamique du calcul dans le cloud, les utilisateurs ayant la possibilité de contrôler les ressources de calcul, de mémoire et de stockage utilisées.
 
-Utilisez un [Bloc-notes Jupyter](https://jupyter.org/) pour créer et partager
-des documents interactifs qui contiennent un mélange de code en direct, de
-visualisations et de texte. Ceux-ci peuvent être écrits en `Python`,` Julia` ou
-`R`.
+Kubeflow simplifie les tâches suivantes :
 
-![Bloc-notes Jupyter](images/jupyter_in_action.png)
+- Création d'environnements personnalisables pour travailler avec des données avec un provisionnement de ressources contrôlé par l'utilisateur (CPU, GPU, RAM et stockage personnalisés).
+- Gestion des serveurs d'ordinateurs portables, y compris Ubuntu Desktop (via noVNC), R Studio, JupyterLab avec Python, R, Julia et SAS pour les employés de Statistique Canada.
 
-Pour lancer un serveur bloc-notes avec une interface Jupyter, choisissez l'une
-des images `jupyterlab` lors de la création de votre serveur bloc-notes. L'image
-`jupyterlab` est également préchargée avec VS Code dans le navigateur si vous
-préférez une expérience IDE complète.
+<!-- prettier-ignore -->
+!!! info "Tableau de bord Kubeflow"
+     - [**Kubeflow Dashboard**](https://kubeflow.aaw.cloud.statcan.ca/) Utilisez ce lien une fois que vous avez votre compte cloud !
 
-### RStudio pour `R` et `Shiny`
+La prise en main de l'Advanced Analytics Workspace (AAW) est simple et rapide. Tout d'abord, vous voudrez vous connecter à Kubeflow pour créer votre premier serveur de bloc-notes exécutant JupyterLab, RStudio ou Ubuntu Desktop. Nous vous encourageons à rejoindre notre chaîne Slack pour entrer en contact avec d'autres data scientists et analystes, poser des questions et partager vos expériences avec la plateforme AAW.
```

Comment:
Change "Advanced Analytics Workspace (AAW)" to espace de travail d'analyse avancée (ETAA)
Change AAW to ETAA